README.rst

```diff
@@ -27,25 +27,28 @@ First, you connect to Scrapinghub::
 You can list the projects available to your account::
 
     >>> conn.project_ids()
-    [u'123', u'456']
+    [123, 456]
 
 And select a particular project to work with::
 
-    >>> project = conn['123']
+    >>> project = conn[123]
```

Comment:
@eliasdorneles they do work, but since the `conn.project_ids()` returns a list of int, it looks more consistent if we use `conn[123]` rather than `conn['123']` in the doc.